src/main.ts

```diff
@@ -25,7 +25,11 @@ async function run() {
     if (config.input_files) {
       const patterns = unmatchedPatterns(config.input_files);
       patterns.forEach((pattern) => {
-        throw new Error(`⚠️  Pattern '${pattern}' does not match any files.`);
+        if (config.input_fail_on_unmatched_files) {
+          throw new Error(`⚠️  Pattern '${pattern}' does not match any files.`);
+        } else {
+          console.warn(`🤔 Pattern '${pattern}' does not match any files.`);
+        }
```

Comment:
When `false` (the default) and no files are matched, the action still fails on https://github.com/softprops/action-gh-release/blob/1ce812a7bd7fa6c191ed3334eeace31579a98f25/src/main.ts#L70

Reply:
this should now be fixed in the latest v2 tag